waspc/cli/Command/Start.hs

```diff
@@ -36,16 +39,21 @@ start = do
     --   Then, next time, we give it data we have about last installation, and it uses that
     --   to decide if installation needs to happen or not. If it happens, it returnes new data again.
     --   Right now we have setup/installation being called, but it has not support for being "smart" yet.
-    liftIO $ putStrLn "Setting up generated project..."
-    setupResult <- liftIO $ Lib.setup outDir options
+    waspSays "Setting up generated project..."
+    setupResult <- liftIO $ Lib.setup outDir
     case setupResult of
         Left setupError -> throwError $ CommandError $ "Setup failed: " ++ setupError
-        Right () -> liftIO $ putStrLn "Setup successful."
+        Right () -> waspSays "Setup successful.\n"
 
     -- TODO: Check node version and then run `npm start` on both web and server.
     --   Again, this is something that Generator should be responsible for, since it knows how the code is generated.
     --   It should tell us how to start stuff and we just start it. It should even do composing of the outputs,
     --   since it knows more than us about that.
+    waspSays "Starting up generated project..."
+    startResult <- liftIO $ Lib.start outDir
+    case startResult of
+        Left startError -> throwError $ CommandError $ "Start failed: " ++ startError
+        Right () -> error "This should never happen, start should never end."
```

Comment:
I just realized how deep this is -> "start should never end" :D.

Reply:
:D